README.md

```diff
@@ -209,6 +210,19 @@ $ curl localhost:8080/v2/user/marginInfo
 }
 ```
 
+## Get Leverage
+Returns the account leverage
+
+```bash
+$ curl localhost:8080/v2/leverage
+```
+
+**Response**
+
+```json
+0.094489
```

Comment:
for consistency it should be JSON formatted

```suggestion
{"leverage":0.094489}
```

Reply:
Done